caddyconfig/httpcaddyfile/directives.go

```diff
@@ -119,6 +119,7 @@ type Helper struct {
 	matcherDefs  map[string]caddy.ModuleMap
 	parentBlock  caddyfile.ServerBlock
 	groupCounter counter
+	state        map[string]interface{}
```

Comment:
Maybe this should be exported so (external/third-party) plugins can also use it.

Reply:
Done